docs/cookbook/images/images.rst

```diff
@@ -16,6 +16,10 @@ There are three places in the Sylius platform where the configuration for images
 
 These configs provide you with a set of filters for resizing images to **thumbnails**.
 
++-----------------------------------------+------------------+
+| ``sylius_admin_product_large_thumbnail``| size: [550, 412] |
```

Comment:
I guess `sylius_admin_product_original` and `sylius_admin_user_avatar_thumbnail` is missing in this table.